doc/use_azure_pipeline_to_run_test.md

```diff
@@ -0,0 +1,43 @@
+# Use Azure Pipelines to Run Tests
+
+This tutorial shows how to use Azure Pipelines to test Azure CLI commands. The usage may be changed in the future.
```

Comment:
worth to highlight the pipeline is used for live test

Reply:
OK